migration/sql-files/084-type-groups-and-child-types.sql

```diff
@@ -39,4 +39,8 @@ CREATE TABLE work_item_child_types (
     position integer DEFAULT 0 NOT NULL
 );
 
-CREATE UNIQUE INDEX work_item_child_types_uidx ON work_item_child_types (parent_work_item_type_id, child_work_item_type_id) WHERE deleted_at IS NULL;
\ No newline at end of file
+CREATE UNIQUE INDEX work_item_child_types_uidx ON work_item_child_types (parent_work_item_type_id, child_work_item_type_id) WHERE deleted_at IS NULL;
```

Comment:
before creating new index do we need to run the script to check uniqueness in existing data, I am sure that we will not have bad data in production for WILT but still safety measure.

Reply:
Good point @pranavgore09

Reply:
Actually no, this is a new table that I introduce and by definition there cannot be any entry. @pranavgore09